spec/person_spec.rb

```diff
@@ -0,0 +1,37 @@
+require_relative '../person'
+
+describe Person do
+  context 'test person class' do
+    it 'This test should return the exact data' do
+      person = Person.new('Najib', 22)
+      age = person.age
+      name = person.name
+      expect(age).to eq(22)
+      expect(name).to eq('Najib')
+    end
+
+    it 'Should test if a person can use services?' do
+      person = Person.new('Najib', 22, parent_permission: true)
+      expect(person.can_use_services?).to eq(true)
+    end
+
+    it 'Should test if a person can use services? under age of 18' do
+      person = Person.new('Najib', 12, parent_permission: false)
+      expect(person.can_use_services?).to eq(false)
+    end
+
+    it 'should test if the name of a person is trimmed?' do
+      person = Person.new('Najib', 22, parent_permission: true)
+      correctname = person.correct_name
+      expect(correctname.length <= 10).to be true
+    end
+
+    it "should add a rental to a person's rentals list" do
+      person = Person.new('Najib', 22, parent_permission: true)
+      book = Book.new('Book Title', 'Author')
+      date = Date.new(2023, 9, 30)
+      person.add_rental(book, date)
+      expect(person.rentals).to include(an_instance_of(Rental))
+    end
+  end
```

Comment:
- [ ] Well done 👍🏻 @najibullahjafari and @MasumaJaffery , you both did a great job by implementing the changes asked by your previous code reviewer. Even the things seem correct when I run the test for the `spec` folder. But I tried to run to the `individual` file i.e. `person_spec` file. Your one test is failing the test just because you didn't add the `Date` library and because of that I got below given screenshot error. 

![Screenshot from 2023-09-22 04-52-42](https://github.com/najibullahjafari/oop-school-library/assets/66476329/e59b7f50-29f6-4bcb-9926-caf27efe31cd)


### Solution

**Please add the `require 'date'` at the top of the file and your problem will be resolved, if you want to verify that you can use the below command** 
```sh 
rspec spec/person_spec.rb
```